files/galaxy/tpv/tools.yml

```diff
@@ -474,10 +474,15 @@ tools:
 
   toolshed.g2.bx.psu.edu/repos/bgruening/whisperx/whisperx/.*:
     inherits: basic_docker_tool
+    gpus: 1
     cores: 1
     mem: 5
+    params:
+      docker_run_extra_arguments: '--gpus all --env WHISPERX_MODEL_DIR=$WHISPERX_MODEL_DIR --env WHISPERX_DEVICE=$WHISPERX_DEVICE'
     env:
       WHISPERX_MODEL_DIR: "/data/db/models/whisperx/whisperx_models/"
+      WHISPERX_DEVICE: "cuda"
+      WHISPERX_COMPUTE_TYPE: "float16"
```

Comment:
It is still needed to add `--env WHISPERX_COMPUTE_TYPE=$WHISPERX_COMPUTE_TYPE` to `docker_run_extra_arguments`, this statement alone does not set the environment variable within the Docker container (try launching a job using version `toolshed.g2.bx.psu.edu/repos/bgruening/whisperx/whisperx/3.4.2+galaxy0`).